README.md

```diff
@@ -19,37 +19,55 @@ RKE2 is a fully compliant Kubernetes distribution, with the following changes:
 
 ## Installation
 
-RKE2 comes with a convenient and configurable installation script that handles OS configuration, binary release download, and installation of an openrc init script or systemd unit file. 
+RKE2 comes with a convenient installation script that will, by default, install via `yum` on RPM-based systems while unpacking the tarball bundle to `/usr/local` on all others.
 
 ### Quick Start
 
-The `install.sh` script provides a convenient way for installing rke2 server and agent, to install rke2 service:
+To install via the default mechanism for your system:
 
 ```sh
-INSTALL_RKE2_VERSION=v0.0.1-alpha.7 ./install.sh
+# Y.O.L.O.
```

Comment:
Sorry to turn down the fun dial - but do we really want our public docs to say YOLO?

Reply:
Well. It is important to provide a simple, scriptable one-liner but it is also important to acknowledge that it is insecure.

Reply:
Its removed now.

Reply:
Thanks

Reply:
eh, im with david unfortunately. this is the source-of-truth for RFed's users/customers. theyre being actively driven to it and can't guarantee everyone will react positively to this. :-/